document/core/exec/instructions.rst

```diff
@@ -689,6 +691,260 @@ Table Instructions
    \end{array}
 
 
+.. _exec-table.copy:
+
+:math:`\TABLECOPY`
+..................
+
+.. todo:: TODO: multi tables
```

Comment:
And also new bounds checking semantics, though I assume that that comes in via the bulk memory spec.

Reply:
Yes, I'll need to rebase again after this. :(